src/components/DateTimePicker/DateTimePicker.tsx

```diff
@@ -1,9 +1,15 @@
 import React, { ReactNode } from 'react'
-import DatePicker from 'react-datepicker'
+import DatePicker, { registerLocale, setDefaultLocale } from 'react-datepicker'
 import InputGroup from 'react-bootstrap/InputGroup'
+import { enUS, ptBR } from 'date-fns/locale'
 import { Icon } from '../Icon'
 import 'react-datepicker/dist/react-datepicker.css'
 
+registerLocale('enUS', enUS)
+registerLocale('ptBR', ptBR)
```

Comment:
Why adding just BR locale? @ocBruno we might open a new issue to discuss about which locales to support. What do you think?

Reply:
Definetly agree @fox1t, I had began adding the rest of them but then I wasn't exactly sure  considering the language differences of the same place of origins and how many to add. 
I think depending on the language difference the resources will need to updated accordingly, so adding language support for en-us en-gb might not be worth it for the time being considering the small differences but other languages such as pt-BR pt-PT vary a bit more.